prql-compiler/Cargo.toml

```diff
@@ -39,10 +36,6 @@ sqlformat = "0.2.0"
 sqlparser = {version = "0.30.0", features = ["serde"]}
 strum = {version = "0.24.0", features = ["std", "derive"]}# for converting enum variants to string
 strum_macros = "0.24.0"
-
-# These bring in `errno`, which fails under wasm. We disable the cli in wasm
-# anyway, so they're not required.
-[target.'cfg(not(target_family="wasm"))'.dependencies]
```

Comment:
I did so much yak shaving for this a month or so ago; seems like they changed the dependency tree so it's no longer a problem. Thanks for removing.

Reply:
I now get why it could be removed — because `prqlc` wasn't being compiled with the rest of the workspace...